fakecam/fake.py

```diff
@@ -86,6 +86,8 @@ def __init__(
         width: int,
         height: int,
         scale_factor: float,
+        no_background: bool,
```

Comment:
To stay in sync with the existing flag, would it not be better to use `use_background` instead?
```suggestion:-0+2
        background_blur: int,
        use_background: bool,
```